javascripts/discourse/components/tag-topics.js

```diff
@@ -0,0 +1,29 @@
+import Category from "discourse/models/category";
+import Component from "@glimmer/component";
+import { ajax } from "discourse/lib/ajax";
+import { tracked } from "@glimmer/tracking";
+
+export default class TagTopics extends Component {
+  @tracked tagtopics = null;
+
+  constructor() {
+    super(...arguments);
+    const count = this.args?.params?.count || 5;
+    const tagtopics = "/tag/" + this.args?.params?.tag + ".json";
```

Comment:
This constant should maybe be called `tagTopicsUrl`, that way it avoids confusion with `this.tagtopics`.

Reply:
Thank you for catching this! I used to call it `featured-tag` but changed it to `tag-topics` to match the syntax of `category-topics` but I totally forgot about addresses this variable conflict